priv/schemas/bundle_config_schema_v3.yaml

```diff
@@ -0,0 +1,103 @@
+---
+"$schema": http://json-schema.org/draft-04/schema#
+title: Bundle Config v3
+description: A config schema for bundles
+type: object
+required:
+  - cog_bundle_version
+  - name
+  - version
+  - commands
+additionalProperties: false
+properties:
+  cog_bundle_version:
+    type: number
+    enum:
+      - 2
```

Comment:
This seemed a little weird. Are we running v2 configs through the v3 schema after upgrading?